src/litserve/loops/base.py

```diff
@@ -140,8 +139,6 @@ async def schedule_task(
         lit_api: LitAPI,
         lit_spec: Optional[LitSpec],
         request_queue: Queue,
-        max_batch_size: int,
-        batch_timeout: float,
```

Comment:
this seems like breaking change and several places bellow

Reply:
this is internal API and not exposed to user directly.